core/src/apps/bitcoin/sign_tx/__init__.py

```diff
@@ -70,7 +71,13 @@ async def sign_tx(
         if coin.decred:
             signer_class = decred.Decred
         elif coin.overwintered:
-            signer_class = zcash.Zcashlike
+            if (
+                coin.coin_name in ["Zcash", "Zcash Testnet"] and
+                msg.version == 5  # remove this condition after Zcash NU5 mainnet activation
```

Comment:
@matejcik and I are inclined to choosing the signer based on `msg.version` only, i.e. ignore the coin name. Based on past experience we can expect the other Zcashlike coins to adopt NU5 sooner or later.

Reply:
https://github.com/trezor/trezor-firmware/pull/2166/commits/4b94cad001411de4a84864be9fa678c53f74025d